musicname/ze_doorhug_and_solo_v6_3.cfg

```diff
@@ -0,0 +1,19 @@
+"music"
+{
+    "boss_1.mp3"                        "Silent Hill 2 OST - White Noiz"
+    "chepoff_l1new.mp3"                 "Two Steps From Hell - To Glory (Minas Tirith)"
+    "du_hast2.mp3"                      "Radio Tapok - Du Hast"
+    "gaga_dog.mp3"                      "Lagy Gaga - Poker Face"
+    "heyey_l1new.mp3"                   "slackcircus - Fabulous Secret Powers"
+    "hobbits_l1new.mp3"                 "Taking The Hobbits to Isengard (Remix)"
+    "kolbasernew.mp3"                   "XS Project - Bochka, Bass, Kolbaser"
+  //"kroti.mp3"                         ""
+    "music_minas.mp3"                   "Pendulum - Encoder"
+    "navras.mp3"                        "Matrix Revolutions OST - Navras"
+    "nigger_l1new.mp3"                  "Dank Memes - Swiggity Swag"
+    "pendulum_autism.mp3"               "Pendulum - Tarantula"
+    "predator - finalmusic - 2.mp3"     "Halo 3 OST - Finish the Fight"
+    "titanic_l1new.mp3"                 "My Heart Will Go On (Recorder)"
```

Comment:
No artist for `titanic_l1new.mp3`?

Reply:
https://www.youtube.com/watch?v=X2WH8mHJnhM
its the recorder meme, not sure if / who should be put down considering this has 2? and all the others are just reposts of the meme

Reply:
The latter.